backend/app/services/collections/create_collection.py

```
@@ -163,29 +134,32 @@ def _mark_job_failed(
 
 def execute_job(
     request: dict,
+    with_assistant: bool,
     project_id: int,
     organization_id: int,
     task_id: str,
     job_id: str,
-    with_assistant: bool,
     task_instance,
```

Ruff (0.14.13)

Unused function argument: `task_instance`

(ARG001)